chef/roles/bmc-nat-router.rb

```diff
@@ -1,7 +1,7 @@
 name "bmc-nat-router"
 description "Configures a node to nat to the BMC network"
 run_list(
-         "recipe[bmc-nat::router]"
+         "recipe[ipmi::role_bmc_nat_router]"
```

Comment:
Style/FirstParameterIndentation: Indent the first parameter one step more than the start of the previous line.